### PR TITLE
User settings

### PR DESCRIPTION
**Pull request by chrizzor**
Adds user settings to the frontend manager.

### CODE REVIEW

File: Resources/Private/Templates/User/Settings.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    the terms of the GNU General Public License, either version 2
+    of the License, or any later version.
+
+    For the full copyright and license information, please read the
+    LICENSE.txt file that was distributed with this source code.
+
+    The TYPO3 project - inspiring people to share!
+    -->
+</f:comment>
+{namespace dpf = EWW\Dpf\ViewHelpers}
```
Comment (claussni) [inline Resources/Private/Templates/User/Settings.html:15]: This template seems to have no translation.

**Comment by chrizzor** [comment]
> Like #175 and #176 activating the extension crashes TYPO3.
> 
> Since the changes in this particular PR are not massive, I suppose there must be something off with the Docker setup. Can you please try to reproduce this?
> 
> I took the following steps to run the containers:
> 
> 1. `$ docker-compose -f Build/docker-compose.yml up` to start the containers
> 2. Open `http://localhost` to set up the TYPO3 system and database to an empty state.
> 3. Go to the extension manager and active `dpf`.

In my case the extension is installable without problems.
Same on the other PR branches.

File: Resources/Private/Templates/User/Settings.html (hunk 1 above)
Comment (chrizzor) [reply Resources/Private/Templates/User/Settings.html:15]: Translations added.

**Comment by chrizzor** [comment]
Branch is now rebased against master

**Comment by chrizzor** [comment]
Branch is now rebased against master

**Pull request merged**